TicTacToeApp/ViewController.swift

```diff
@@ -8,18 +8,38 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+public class ViewController: UIViewController {
+  var currentPlayerMark = "X"
 
-  override func viewDidLoad() {
+  override public func viewDidLoad() {
     super.viewDidLoad()
     // Do any additional setup after loading the view, typically from a nib.
```

Comment:
I like to keep the boilerplate around for unfamiliar frameworks until I know it's not relevant, because it seems like a strong hint of the kind of design that the framework encourages - but yeah, I understand. I will henceforth not commit comments.